linodecli/plugins/obj/__init__.py

```diff
@@ -486,9 +487,8 @@ def print_help(parser: ArgumentParser):
         for name, func in sorted(COMMAND_MAP.items())
     ]
 
-    tab = SingleTable(command_help_map)
-    tab.inner_heading_row_border = False
-    print(tab.table)
+    tab = Table(show_header=False)
+    rprint(tab)
```

Comment:
This should take care of that failed `pylint` check blocking the merge. I went ahead and

- Made these changes locally
- Checked to make sure the help table populates with `linode-cli obj --help`
- Ran `pylint linodecli` in the root project dir
- For good measure, checked the rest of the code base where the new `rich.Table` objects are to made sure their row content was added  

Should be all good to go now. Please commit this suggestion if you agree with the changes.

```suggestion
    for row in command_help_map:
        tab.add_row(*row)
    rprint(tab)
```

Reply:
Committed so you can just re-run the checks if you desire.